server/controllers/refer/referralsController.test.ts

```diff
@@ -236,34 +226,11 @@ describe('ReferralsController', () => {
       courseService.getCourse.mockResolvedValue(course)
       ;(CourseUtils.presentCourse as jest.Mock).mockReturnValue(coursePresenter)
 
-      const addedByUser = userFactory.build()
-
-      const earliestCourseParticipationPresenter: CourseParticipationPresenter = {
-        ...courseParticipationFactory.build({
-          addedBy: addedByUser.username,
-          createdAt: '2022-01-01T12:00:00.000Z',
-        }),
-        addedByDisplayName: StringUtils.convertToTitleCase(addedByUser.name),
-      }
-
-      const latestCourseParticipationPresenter: CourseParticipationPresenter = {
-        ...courseParticipationFactory.build({
-          addedBy: addedByUser.username,
-          createdAt: '2023-01-01T12:00:00.000Z',
-        }),
-        addedByDisplayName: StringUtils.convertToTitleCase(addedByUser.name),
-      }
-
-      const courseParticipationPresenters = [earliestCourseParticipationPresenter, latestCourseParticipationPresenter]
-
-      courseService.getAndPresentParticipationsByPrisonNumber.mockResolvedValue(courseParticipationPresenters)
-
-      const summaryListOptions = 'summary list options'
-      ;(CourseParticipationUtils.summaryListOptions as jest.Mock).mockImplementation(
-        (_courseParticipation, _referralId, _withActions = { change: true, remove: true }) => {
-          return summaryListOptions
-        },
-      )
+      const summaryListOptions = 'summary list options' as unknown as GovukFrontendSummaryListWithRowsWithValues
```

Comment:
😬

Reply:
Yeah, it's a bit ugly, but better than defining a whole summary list I think!